src/rules/semicolonRule.ts

```diff
@@ -58,6 +58,8 @@ export class Rule extends Lint.Rules.AbstractRule {
 
     public static FAILURE_STRING_MISSING = "Missing semicolon";
     public static FAILURE_STRING_UNNECESSARY = "Unnecessary semicolon";
+    public static FIX_STRING_MISSING = "Add semicolon";
```

Comment:
are fix strings necessary for your use case? It seems to me that the failure string would be an apt descriptor in most cases